src/components/JoinRequests/JoinRequestsTable/utils.tsx

```diff
@@ -14,13 +18,24 @@ export const getSortIcon = ({
   );
 };
 
-export const getRequestsFromData = <T,>(data: T[]): JSX.Element => {
+export const getRequestsFromData = <T,>(data: T[], typeText: string): JSX.Element => {
```

Comment:
For typeText, use the enum as the type instead of string. I'm trying to implement this everywhere. So `InstitutionType.SCHOOL` for eg